sources/packages/backend/apps/api/src/route-controllers/institution/institution.controller.service.ts

```diff
@@ -80,4 +80,15 @@ export class InstitutionControllerService {
       description: institutionType.name,
     }));
   }
+
+  /**
+   * Check if institution is private or public.
+   * @param institutionId is the institution id.
+   * @returns boolean true if institution is private, else false.
+   */
+  async checkIfInstitutionIsPrivate(institutionId: number): Promise<boolean> {
```

Comment:
I believe that this method can be moved to `api\src\services\institution\institution.service.ts`.